src/worldstate/routine/include/State.h

```diff
@@ -0,0 +1,32 @@
+//
+// Created by joel on 06/05/18.
+//
+
+#ifndef PROJECT_ROUTINE_H
+#define PROJECT_ROUTINE_H
+
+#include <ros/ros.h>
+#include <worldstate/state_msg.h>
+/*
+ * Documentation to Provide
```

Comment:
Took me a couple minutes to understand that this was an instruction. Should replace with something like "Implementers of the State class should provide the following documentation:"